components/process.rst

```diff
@@ -100,6 +100,32 @@ with a non-zero code)::
         echo $exception->getMessage();
     }
 
+.. tip::
+
+    .. versionadded:: 3.3
+        The ability to define commands as arrays of arguments was introduced in
+        Symfony 3.3.
+
+    Defining commands as arrays of arguments is recommended because the Process
+    component escapes those arguments automatically. Defining commands as
```

Comment:
"Automatically escaping" does not seem to be the right wording according to your comment in https://github.com/symfony/symfony/issues/27796#issuecomment-401703541 😄

Reply:
reworded :)